### ISSUE TITLE
Cell::TemplateMissingError while starting rails server as a daemon

### ISSUE THREAD

**Issue by MrShemek** [trailblazer/cells-rails#17]
Hi there,

I noticed a strange thing. Everything works fine when I start rails server "normally":
`rails s -p 5002 -b 0.0.0.0`

But when I start is as a daemon:
`rails s -p 5002 -b 0.0.0.0 -d`

I get the following error:
```
Cell::TemplateMissingError in Admin::Home#index
Showing /Users/MrShemek/Sites/TH/app/views/admin/home/index.html.erb where line #4 raised:

Template missing: view: `checklist.erb` prefixes: ["app/cells/admin_setup"]

Extracted source (around line #9):
  def checklist
    check_steps
    render if should_render?
  end

  private
```

Line `#9` is `render if should_render?`.

I was able to replicate this issue with new Rails instance (`rails` + `cells-rails` + `cells-erb`).

My envrionement:
rails 5.0.0.1
cells 4.1.5
cells-rails 0.0.6
cells-erb 0.0.9

**Comment by andrelinoge** [comment]
Also has this issue
ruby 2.3.4
rails 4.2.8
cells 4.1.7
cells-rails 0.0.8
cells-slim 0.0.5

**Comment by apotonick** [comment]
Are you actually using ERB?

**Comment by art-solopov** [comment]
Witnessing the same issue.

* Ruby 2.4.3
* Rails 5.1.4
* Cells 4.1.7
* Cells-rails 0.0.8
* Cells-erb 0.1.0

**Comment by sachins1812** [comment]
I am also facing the same error please let me know which gem is required


`Cell::TemplateMissingError at /clients/94/orders/mass_item_export
Template missing: view: `show.erb` prefixes: ["/home/pc/projects/portal/app/cells/exports/export"]`

**Comment by MrShemek** [comment]
@sachins1812 it was a long time ago but due to few problems and lack of support I stopped using Cells gem and implement the functionality by myself.

**Comment by apotonick** [comment]
Great @MrShemek that's the spirit of Open-Source! :joy:

**Comment by MrShemek** [comment]
@apotonick and what did you do with that issue in the last 3.5 years?

**Comment by apotonick** [comment]
It didn't affect me - in fact, it's really hard to track down issues like that if you can't provoke the bug yourself! You can
a) copy the functionality and throw away the gem (that's what happened)
b) set up a test case for the developers of the gem so they can fix it for you, without wasting hours of their life for something that doesn't bother them
c) do b) plus jump into the code and try to assist. Cells has 300-400 LoC, it's super simple, I'm pretty sure you would've spotted the problem yourself (if you can rewrite Cells yourself you most probably would've found this bug).

Regarding my 3.5 years, those are documented here: https://trailblazer.to/2.1/blog.html#blog-official-2-1-release-notes

Let me know if you want to join the bug hunt! :beers:

**Comment by MrShemek** [comment]
No one said I've rewrote Cells - I just implemented something super simple 😄  It was at the beginning of Ruby career and that was the best solution at that time.

Even if I am no longer using this gem, I will try to replicate the issue once I have time. My old project does not exist anymore, so anyone who has this issue, please share the code to make it easier.

**Comment by apotonick** [comment]
Isn't that funny? Now I'm facing that very problem *myself*!!! :laughing:

**Comment by apotonick** [comment]
The problem is in the `trailblazer-cells` gem, not in this gem! We will release a fix shortly.

**Comment by apotonick** [comment]
This annoying bug is finally fixed in `cells-rails-0.1.1`: https://github.com/trailblazer/cells-rails/commit/1aa3f67bea9b5545fbee7f9c6114153f9119faeb

Sorry to y'all for taking so long! :heart:
